assert/assertions.go

```diff
@@ -246,7 +248,7 @@ func Fail(t TestingT, failureMessage string, msgAndArgs ...interface{}) bool {
 
 // Implements asserts that an object is implemented by the specified interface.
 //
-//    assert.Implements(t, (*MyInterface)(nil), new(MyObject), "MyObject")
```

Comment:
Why are the messages removed in the examples? I'd expect the methods without the `f` suffix to still accept arguments which are appended similar to how `fmt.Println` does.